jobs/resource_job.go

```diff
@@ -406,10 +432,20 @@ func jobSettingsSchema(s *map[string]*schema.Schema, prefix string) {
 	}
 }
 
+func gitSourceSchema(s *schema.Resource, prefix string) {
+	if p, err := common.SchemaPath(s.Schema, prefix, "url"); err == nil {
```

Comment:
prefix is always an empty string, so why wouldn't you just `s["url"].ValidateFunc = validation.IsURLWithHTTPS`?...